<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Road Designer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewTerrain"/>
    <addaction name="actionOpenTerrain"/>
    <addaction name="actionSaveTerrain"/>
    <addaction name="separator"/>
    <addaction name="actionLoadRoads"/>
    <addaction name="actionSaveRoads"/>
    <addaction name="actionClearRoads"/>
    <addaction name="separator"/>
    <addaction name="actionLoadAreas"/>
    <addaction name="actionSaveAreas"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionControlWidget"/>
    <addaction name="actionPropertyWidget"/>
   </widget>
   <widget class="QMenu" name="menuArea">
    <property name="title">
     <string>Area</string>
    </property>
    <addaction name="actionAreaSelect"/>
    <addaction name="actionAreaCreate"/>
    <addaction name="actionAreaDelete"/>
    <addaction name="actionHintLine"/>
    <addaction name="separator"/>
    <addaction name="actionHighwaySketch"/>
    <addaction name="actionAvenueSketch"/>
    <addaction name="separator"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuPM">
    <property name="title">
     <string>PM</string>
    </property>
    <addaction name="actionGenerateBlocks"/>
    <addaction name="actionGenerateParcels"/>
    <addaction name="actionGenerateBuildings"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuRoads">
     <property name="title">
      <string>Roads</string>
     </property>
     <addaction name="actionDisplayHighway"/>
     <addaction name="actionDisplayBoulevard"/>
     <addaction name="actionDisplayAvenue"/>
     <addaction name="actionDisplayLocalStreet"/>
    </widget>
    <addaction name="menuRoads"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRenderingDefault"/>
    <addaction name="actionRenderingTexture"/>
    <addaction name="actionRenderingGroupBy"/>
    <addaction name="actionRenderingGenerationType"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuArea"/>
   <addaction name="menuPM"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuRendering"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="areaToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewTerrain">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/New_24x24.png</normaloff>:/MainWindow/Resources/New_24x24.png</iconset>
   </property>
   <property name="text">
    <string>New Terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenTerrain">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Open_24x24.png</normaloff>:/MainWindow/Resources/Open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Open Terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionControlWidget">
   <property name="text">
    <string>Control Widget</string>
   </property>
  </action>
  <action name="actionDeleteEdge">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Delete_24x24.png</normaloff>:/MainWindow/Resources/Delete_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Delete Edge</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSaveTerrain">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Save_24x24.png</normaloff>:/MainWindow/Resources/Save_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Save Terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Undo_24x24.png</normaloff>:/MainWindow/Resources/Undo_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Cut_24x24.png</normaloff>:/MainWindow/Resources/Cut_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Copy_24x24.png</normaloff>:/MainWindow/Resources/Copy_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Paste_24x24.png</normaloff>:/MainWindow/Resources/Paste_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionOpenToAdd">
   <property name="text">
    <string>Open To Add</string>
   </property>
  </action>
  <action name="actionVoronoi">
   <property name="text">
    <string>Voronoi</string>
   </property>
  </action>
  <action name="actionVoronoiCut">
   <property name="text">
    <string>Voronoi Cut</string>
   </property>
  </action>
  <action name="actionModeBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionModeLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionModeSketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sketch</string>
   </property>
  </action>
  <action name="actionShowArea">
   <property name="text">
    <string>Show Area</string>
   </property>
  </action>
  <action name="actionAreaSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Select_24x24.png</normaloff>:/MainWindow/Resources/Select_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionAreaCreate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Create_24x24.png</normaloff>:/MainWindow/Resources/Create_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionHighwaySketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/SketchH_24x24.png</normaloff>Resources/SketchH_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Highway Sketch</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveRoads">
   <property name="text">
    <string>Save Roads</string>
   </property>
  </action>
  <action name="actionLoad_Highways">
   <property name="text">
    <string>Load Highways</string>
   </property>
  </action>
  <action name="actionSave_Highways">
   <property name="text">
    <string>Save Highways</string>
   </property>
  </action>
  <action name="actionAvenueSketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/SketchB_24x24.png</normaloff>:/MainWindow/Resources/SketchB_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Avenue Sketch</string>
   </property>
  </action>
  <action name="actionPropertyWidget">
   <property name="text">
    <string>Property Widget</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Debug_24x24.png</normaloff>:/MainWindow/Resources/Debug_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action3DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/3DView_24x24.png</normaloff>:/MainWindow/Resources/3DView_24x24.png</iconset>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Terrain_32x32.png</normaloff>:/MainWindow/Resources/Terrain_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
  </action>
  <action name="actionLoadRoads">
   <property name="text">
    <string>Load Roads</string>
   </property>
  </action>
  <action name="actionLoadAreas">
   <property name="text">
    <string>Load Areas</string>
   </property>
  </action>
  <action name="actionSaveAreas">
   <property name="text">
    <string>Save Areas</string>
   </property>
  </action>
  <action name="actionGenerateBlocks">
   <property name="text">
    <string>Generate Blocks</string>
   </property>
  </action>
  <action name="actionGenerateParcels">
   <property name="text">
    <string>Generate Parcels</string>
   </property>
  </action>
  <action name="actionGenerateBuildings">
   <property name="text">
    <string>Generate Buildings</string>
   </property>
  </action>
  <action name="actionAreaDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClearRoads">
   <property name="text">
    <string>Clear Roads</string>
   </property>
  </action>
  <action name="actionPMControlWidget">
   <property name="text">
    <string>PM Control Widget</string>
   </property>
  </action>
  <action name="actionHighways">
   <property name="text">
    <string>Highway</string>
   </property>
  </action>
  <action name="actionBoulevard">
   <property name="text">
    <string>Boulevard</string>
   </property>
  </action>
  <action name="actionAvenue">
   <property name="text">
    <string>Avenue</string>
   </property>
  </action>
  <action name="actionLocal_Street">
   <property name="text">
    <string>Local Street</string>
   </property>
  </action>
  <action name="actionDisplayHighway">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highway</string>
   </property>
  </action>
  <action name="actionDisplayBoulevard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Boulevard</string>
   </property>
  </action>
  <action name="actionDisplayAvenue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Avenue</string>
   </property>
  </action>
  <action name="actionDisplayLocalStreet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local Street</string>
   </property>
  </action>
  <action name="actionHintLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Line_24x24.png</normaloff>:/MainWindow/Resources/Line_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Hint Line</string>
   </property>
  </action>
  <action name="actionRenderingDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionRenderingTexture">
   <property name="text">
    <string>Texture</string>
   </property>
  </action>
  <action name="actionRenderingGroupBy">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionColorByGroup">
   <property name="text">
    <string>By Group</string>
   </property>
  </action>
  <action name="actionColorDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionRenderingGenerationType">
   <property name="text">
    <string>Generation Type</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
